### DISCUSSION TITLE
Video intro

### DISCUSSION THREAD

**Discussion by mlopezmad** [OnionUI/Onion#1325] [Ideas]
Would it be possible to add a menu to play an intro video when booting the console?

**Comment by schmurtzm** [comment]
That's technically possible but not at the firmware level, just after (at the second boot screen, the one that you can personalize with the theme). However it will just pause the loading process to play a video and we try to have the faster boot possible in Onion.
I suppose that video is interesting only if the loading of the OS continues in background 🤔

**Reply by mlopezmad** [reply]
mmm so if it takes longer to start forget what I said. and I'm going to take the opportunity to combine it with another suggestion. 
Is it possible to make a bezel activation menu? that is, screen to activate or deactivate them, all at the same time. If it can be done, perhaps it could also be developed individually? 
But I would start by activating all the bezels and deactivating them.
Thank you!!

**Reply by mlopezmad** [reply]
If necessary I will open another thread as a suggestion or wish 😅☺️ I think it is very interesting
